turbinia/workers/plaso.py

```diff
@@ -46,76 +92,19 @@ def run(self, evidence, result):
     Returns:
         TurbiniaTaskResult object.
     """
-    config.LoadConfig()
 
-    # TODO: Convert to using real recipes after
-    # https://github.com/google/turbinia/pull/486 is in.  For now we're just
-    # using the --recipe_config flag, and this can be used with colon separated
-    # values like:
-    # --recipe_config='artifact_filters=BrowserFoo:BrowserBar,parsers=foo:bar'
-    if evidence.config and evidence.config.get('artifact_filters'):
-      artifact_filters = evidence.config.get('artifact_filters')
-      artifact_filters = artifact_filters.replace(':', ',')
-    else:
-      artifact_filters = None
-
-    if evidence.config and evidence.config.get('parsers'):
-      parsers = evidence.config.get('parsers')
-      parsers = parsers.replace(':', ',')
-    else:
-      parsers = None
-
-    if evidence.config and evidence.config.get('file_filters'):
-      file_filters = evidence.config.get('file_filters')
-      file_filter_file = os.path.join(self.tmp_dir, 'file_filter.txt')
-      try:
-        with open(file_filter_file, 'wb') as file_filter_fh:
-          for filter_ in file_filters.split(':'):
-            file_filter_fh.write(filter_.encode('utf-8') + b'\n')
-      except IOError as exception:
-        message = 'Cannot write to filter file {0:s}: {1!s}'.format(
-            file_filter_file, exception)
-        result.close(self, success=False, status=message)
-        return result
-    else:
-      file_filters = None
-      file_filter_file = None
-
-    if evidence.config and evidence.config.get('vss'):
-      vss = evidence.config.get('vss')
-    else:
-      vss = None
-
-    if evidence.config and evidence.config.get('yara_rules'):
-      yara_rules = evidence.config.get('yara_rules')
-      with NamedTemporaryFile(dir=self.tmp_dir, delete=False, mode='w') as fh:
-        yara_file_path = fh.name
-        fh.write(yara_rules)
-    else:
-      yara_rules = None
+    config.LoadConfig()
 
     # Write plaso file into tmp_dir because sqlite has issues with some shared
     # filesystems (e.g NFS).
     plaso_file = os.path.join(self.tmp_dir, '{0:s}.plaso'.format(self.id))
     plaso_evidence = PlasoFile(source_path=plaso_file)
     plaso_log = os.path.join(self.output_dir, '{0:s}.log'.format(self.id))
 
-    # TODO(aarontp): Move these flags into a recipe
-    cmd = (
-        'log2timeline.py --status_view none --hashers all '
-        '--partition all').split()
-    if config.DEBUG_TASKS or evidence.config.get('debug_tasks'):
+    cmd = self.build_plaso_command('log2timeline.py', self.task_config)
+
+    if self.task_config['debug_tasks']:
```

Comment:
I think here we still want to check `config.DEBUG_TASKS` to see if this has been hard-coded in the config.